packages/serverless-helpers/src/helpers/index.ts

```diff
@@ -1,4 +1,3 @@
-export { applyHttpMiddlewares } from './applyHttpMiddlewares';
```

Comment:
This commit is breaking, so you need to add a `!` character in the commit message, like so:
`feat(serverless-helpers)!: remove applyHttpMiddlewares helper`